### PR TITLE
feat: add a notice for gsoc 2021 orgs

### PR DESCRIPTION
**Pull request by singlamohit1**

### CODE REVIEW

File: src/components/notification.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { Message } from "semantic-ui-react"
+
+const MessageExampleInfo = () => {
```
Comment (nishantwrp) [inline src/components/notification.jsx:4]: `MessageExampleInfo` ?

File: src/pages/index.jsx [hunk 2]
```diff
@@ -198,6 +199,9 @@ const IndexPage = ({ data }) => {
       homePage={true}
     >
       <SEO title={"Home"} meta={meta} />
+      <Grid className="index-org-cards-grid">
+        <Notification />
+      </Grid>
```
Comment (nishantwrp) [inline src/pages/index.jsx:204]: Do we need `Grid` and `index-org-cards-grid` class?

**Review by nishantwrp — CHANGES_REQUESTED** [review]

File: src/components/notification.jsx (hunk 1 above)
Comment (singlamohit1) [reply src/components/notification.jsx]: done

File: src/pages/index.jsx (hunk 2 above)
Comment (singlamohit1) [reply src/pages/index.jsx:204]: yes

**Review by nishantwrp — APPROVED** [review]

**Pull request merged**